package.json

```diff
@@ -1,8 +1,14 @@
 {
   "name": "fh-sync",
-  "version": "1.0.13",
+  "version": "1.0.14",
```

Comment:
TypeScript community recommends to make minor patches for all non trivial changes in types. 
Many people may have patch version classifiers for types and suddenly their project will stop compiling on master. It happened couple times for us. Best to do minor bump.

Reply:
👍 Good call, will do!